tests/src/common/Wsk.scala

```diff
@@ -226,6 +225,7 @@ class WskAction(override val usePythonCLI: Boolean = false)
         annotations: Map[String, JsValue] = Map(),
         timeout: Option[Duration] = None,
         memory: Option[Int] = None,
+        logsize: Option[Int] = None,
```

Comment:
you could make this take Size instead of Int (since you're using `0 MB` in the test with `.toInt`.